### PR TITLE
Update wiliwili.json

### PR DESCRIPTION
**Pull request by DDinghoya**

### CODE REVIEW

File: resources/i18n/ko/wiliwili.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "wiliwili",
+  "title": "윌리윌리",
```
Comment (xfangfang) [inline resources/i18n/ko/wiliwili.json]: I think it's better to keep the original name of the software

File: resources/i18n/ko/wiliwili.json [hunk 2]
```diff
@@ -97,11 +97,11 @@
   "hints": {
     "hint1": "홈 화면에서 게임 선택",
     "hint2": "\uE0E5 키를 누른 상태에서 \uE0E0 을 클릭하여 hbmenu로 진입",
-    "hint3": "hbmenu에서, wiliwili 선택",
+    "hint3": "hbmenu에서, 윌리윌리 선택",
     "hint4": "선택 사항: 제공된 wiliwili NSP 포워더 사용 (wiliwili.nro를 /switch/wiliwili.nro에 배치)",
     "hint_nsp": "NSP 포워더를 설치하려면 \uE0E0 를 클릭",
     "hint_confirm": "NSP 포워더를 설치하면 차단될 수 있음\n다시 확인 필요",
-    "hint_right": "사진은 인터넷에서 가져온 것이며 저작권은 Nintendo Co., Ltd에 있음"
+    "hint_right": "사진은 인터넷에서 가져온 것이며 저작권은 닌텐도에 있음"
```
Comment (xfangfang) [inline resources/i18n/ko/wiliwili.json]: This should not be translated

File: resources/i18n/ko/wiliwili.json [hunk 3]
```diff
@@ -156,7 +156,7 @@
       "logout": "현재 계정에서 로그아웃 \n(응용 프로그램이 종료)",
       "scan": "QR 코드를 스캔하여 로그인",
       "loading": "QR 코드 로드 중...",
-      "hint": "BiliBili 클라이언트를 사용하여 QR 코드 스캔",
+      "hint": "빌리빌리 클라이언트를 사용하여 QR 코드 스캔",
```
Comment (xfangfang) [inline resources/i18n/ko/wiliwili.json]: This should not be translated

File: resources/i18n/ko/wiliwili.json [hunk 4]
```diff
@@ -55,7 +55,7 @@
       "tab": "정보",
       "brief_header": "소개",
       "repository": "저장소",
-      "brief": "또 다른 타사 bilibili 클라이언트",
+      "brief": "또 다른 타사 빌리빌리 클라이언트",
```
Comment (xfangfang) [inline resources/i18n/ko/wiliwili.json]: This should not be translated

File: resources/i18n/ko/wiliwili.json [hunk 5]
```diff
@@ -97,11 +97,11 @@
   "hints": {
     "hint1": "홈 화면에서 게임 선택",
     "hint2": "\uE0E5 키를 누른 상태에서 \uE0E0 을 클릭하여 hbmenu로 진입",
-    "hint3": "hbmenu에서, wiliwili 선택",
+    "hint3": "hbmenu에서, 윌리윌리 선택",
```
Comment (xfangfang) [inline resources/i18n/ko/wiliwili.json]: This should not be translated

**Review by xfangfang — COMMENTED** [review]
Thank you very much for your continuous attention to this project.

File: resources/i18n/ko/wiliwili.json (hunk 1 above)
Comment (DDinghoya) [reply resources/i18n/ko/wiliwili.json]: Ok, I'll change origin. :)

File: resources/i18n/ko/wiliwili.json (hunk 2 above)
Comment (DDinghoya) [reply resources/i18n/ko/wiliwili.json]: Yup!

File: resources/i18n/ko/wiliwili.json (hunk 3 above)
Comment (DDinghoya) [reply resources/i18n/ko/wiliwili.json]: Yup!

File: resources/i18n/ko/wiliwili.json (hunk 4 above)
Comment (DDinghoya) [reply resources/i18n/ko/wiliwili.json]: Yup!

File: resources/i18n/ko/wiliwili.json (hunk 5 above)
Comment (DDinghoya) [reply resources/i18n/ko/wiliwili.json]: Yup!

**Review by DDinghoya — COMMENTED** [review]
Done

**Comment by DDinghoya** [comment]
There is no problem with Korean output. It has been completed.

I hope that it will be very useful for switch Korean users who understand Chinese. :)

**Pull request merged**